index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE HTML>
+<html lang="en">
+<head>
+	<title>Jordan Addis - Portfolio</title>
+	<!-- Styling -->
+	<link href="style.css" type="text/css" rel="stylesheet">
+	<link href="normalize.css" type="text/css" rel="stylesheet">
+	<!-- Fonts -->
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet"> 
+	<!-- Scripts -->
+	<script src="https://kit.fontawesome.com/7c6af540e7.js" crossorigin="anonymous"></script>
+</head>
+<body>
+	<a class="anchor" id="top"></a>
+	<nav>
+		<div class="navLeft">
+			<a href="#top" class="navLogo">
+				<h3>Jordan Addis</h3>
+			</a>
+		</div>
+		<div class="navRight">
+			<a href="#top" class="navLink">Home</a>
+			<a href="#about" class="navLink">About Me</a>
+			<a href="#projects" class="navLink">Projects</a>
+			<a href="#contact" class="navLink">Contact</a>
+		</div>
+		<a href="#" class="burgerMenu">
+				<div></div>
+				<div></div>
+				<div></div>
+		</a>
+	</nav>
+	<header>
+		<div class="headerContent">
+			<div class="intro">
+				<h1 class="headerTitle">Hi, I'm Jordan</h1>
+				<p class="headerText">I am a fullstack developer in Bristol, UK</p>
+				<div class="headerLinks">
+					<a class="headerBtn" href="mailto:jordanaddis@gmail.com?subject=Let&apos;s&nbsp;work&nbsp;together">Hire me</a>
```

Comment:
Totally my own preference but might also be that of others; is there a way so that it doesn't use the default mailer on the device i.e. if you don't use your desktop email app for messaging and instead use browser or e.g. Outlook?

Reply:
As far as I know, this can only be set in the user's browser preferences